README.md

```diff
@@ -28,6 +28,8 @@ Configuration
 
 * `RETIREMENT_EDX_OAUTH2_CLIENT_ID` (default `"retirement_service_worker"`)
 * `RETIREMENT_COOL_OFF_DAYS` (default `30`)
+* `RETIREMENT_K8S_CRONJOB_SCHEDULE` (default `"0 0 * * *"` once a day at 
```

Comment:
Micro-nit: this probably needs a comma before "once".

Reply:
yes